.github/actions/setup-linux/action.yml

```diff
@@ -59,6 +59,10 @@ runs:
           aws ecr get-login-password --region "$AWS_DEFAULT_REGION" | docker login --username AWS \
               --password-stdin "$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com"
 
+          # For LF Runners we need to make sure we also login to Meta's ECR docker registry too.
```

Comment:
I believe we want to do this in two PRs.

As there is a risk of one of the changes reverted. We might want to merge this one first, and then merge the `_runner-determinator.yml` PR a few days latter

Reply:
Done. Opened #132870 for the docker login part.